sorbet/tapioca/require.rb

```diff
@@ -3,3 +3,4 @@
 
 # Add your extra requires here
 require "rails/all"
+require "activerecord-typedstore"
```

Comment:
There are new line errors for this file and the `shim`.

In my editor I can insert a new line below and delete it and it gets resolved. I didn't try VScode but you should add a new line character at the end of these 2 files.